src/data_exports/write_csv.py

```diff
@@ -0,0 +1,26 @@
+import os
+import glob
+
+def save_spark_dataframe_as_csv(df_output, output_name):
+    '''
+    Function to save spark dataframe saved within the outputs dictionary as a csv
+    '''
```

Comment:
The docstring could do with some expansion with the details about what the function does, as well as the inputs and outputs